tests/repl/tests.rs

```diff
@@ -25,7 +25,7 @@ pub fn run_test(input: &str, expected: &str) -> TestResult {
     let mut file = NamedTempFile::new()?;
     let name = file.path();
 
-    let mut cmd = Command::cargo_bin("nu")?;
+    let mut cmd: Command = Command::cargo_bin("nu")?;
```

Comment:
Is this working around a type inference issue?

Reply:
Not meant to be there - sorry. Im not sure why it was added, I will remove it.